automation/taskcluster/lib/tasks.py

```diff
@@ -45,37 +38,27 @@ def build_task(self, name, description, command, artifacts={}, scopes=[], featur
                     command
                 ],
                 "artifacts": artifacts,
-                "deadline": taskcluster.stringDate(deadline)
             },
-            "provisionerId": "aws-provisioner-v1",
-            "metadata": {
-                "name": name,
-                "description": description,
-                "owner": self.owner,
-                "source": self.source
-            }
-        }
+            treeherder={
+                'jobKind': 'build',
+                'machine': {
+                  'platform': 'android-all',
```

Comment:
huh, TIL about this platform

Reply:
I just made it up. Treeherder accepts any string in this field. Providing a special one allows pretty strings.

By the way, if someone has a better name, I'm fine changing it!

Reply:
No,no - the name sounds great to me. Nice to start seeing mobile in the treeherder context! \o/ I guess I'm just surprised we're already living in the glorious future :)